pom.xml

```diff
@@ -1,78 +1,119 @@
 <?xml version='1.0' encoding='UTF-8'?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-   <modelVersion>4.0.0</modelVersion>
-   <parent>
-      <groupId>org.infinispan</groupId>
-      <artifactId>infinispan-persistence-parent</artifactId>
-      <version>9.0.0-SNAPSHOT</version>
-   </parent>
-   <artifactId>infinispan-cachestore-cassandra</artifactId>
-   <packaging>bundle</packaging>
-   <name>Infinispan CassandraCacheStore</name>
-   <description>Infinispan CassandraCacheStore module</description>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
```

Comment:
Could you please reformat it along with our [Infinispan formatter](https://github.com/infinispan/infinispan/tree/master/ide-settings)?